requirements.txt

```diff
@@ -0,0 +1 @@
+ops >= 1.5.2
```

Comment:
Consider updating to v1.5.3 explicitly

Reply:
updated in c7c870362d33e40e70c1d2ff64400dc4c6e33b1b